features/step_definitions/test_steps.rb

```diff
@@ -1,4 +1,4 @@
-Given /^test "(.+)" was created(?: (1 day|[2-9]\d* days) ago)? by (.+) with key (.+) for version (.+) of project (.+)$/ do |name,days,user_name,test_key,project_version,project_name|
+Given /^test "(.+)" was created(?: (\d*) (day|days|week|weeks) ago)? by (.+) with key (.+) for version (.+) of project (.+)$/ do |name,days,interval,user_name,test_key,project_version,project_name|
```

Comment:
It would be simpler to edit if you replaced `(day|days|week|weeks)` by `(day|week)s?`.